test/unit/filters/UserTypeFilterSpec.scala

```diff
@@ -73,6 +73,26 @@ class UserTypeFilterSpec extends ControllerSpec with BeforeAndAfterEach with Aut
       status(result) shouldBe SEE_OTHER
     }
 
+    "return Redirect (303) when an organisation with an assistant account attempts to access a route with existing enrollment" in {
+      val atarEnrolment = Enrolment("HMRC-ATAR-ORG").withIdentifier("EORINumber", "GB134123")
+
+      AuthBuilder.withAuthorisedUser(
+        "user-1236213",
+        mockAuthConnector,
+        userAffinityGroup = AffinityGroup.Organisation,
+        userCredentialRole = Some(Assistant),
+        otherEnrolments = Set(atarEnrolment)
+      )
+
+      val result = controller
+        .download()
```

Comment:
why is this download.apply when the other test cases are startSubscription?

Reply:
Also a good q - and similar answer as above. Most of subscription controllers (e.g. ApplicationController which is the first point of entry for Subscription journey and Controller used for tests here) have the same authorisation checks. 

I don't know why, but for User Type checks this specific controller was used so I felt this could be a good place for additional test even the functionality is tested at `ApplicationControllerSpec.scala`